wagtail/api/v2/filters.py

```diff
@@ -63,8 +63,8 @@ def filter_queryset(self, request, queryset, view):
 class OrderingFilter(BaseFilterBackend):
     def filter_queryset(self, request, queryset, view):
         """
-        This applies ordering to the result set
-        Eg: ?order=title
+        This applies ordering to the result set with support for multiple fields.
+        Eg: ?order=title,created_at
```

Comment:
```suggestion
       Eg: ?order=title or ?order=title,created_at
```

Maybe let's have two examples here.